hw/ip/otbn/doc/_index.md

```diff
@@ -490,6 +490,35 @@ The error that caused the alert can be determined by reading the {{< regref "FAT
 If OTBN was running, this value will also be reflected in the {{< regref "ERR_BITS" >}} register.
 A fatal alert can only be cleared by resetting OTBN through the `rst_ni` line.
 
+
+### Error Prioritization
```

Comment:
A general comment about the language in this section: Is it the error bits we are concerned about, or the errors, which then happen to be represented as a bit in a field? I think it's the errors, so "error bits" should be "errors".